core/src/main/java/org/akaza/openclinica/bean/extract/DownloadDiscrepancyNote.java

```diff
@@ -188,6 +190,167 @@ public int getThreadListContentLength(List<DiscrepancyNoteThread> threadBeans) {
+
+        writer.append(threadNumber+"");
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getId()+"");
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getParentDnId()>0?discNoteBean.getParentDnId():"");
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getCreatedDateString()+"");
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getUpdatedDateString()+"");
+        writer.append(delimiter);
+
+        if (discNoteBean.getParentDnId() == 0){
+            writer.append(discNoteBean.getAge()+"");
+            writer.append(delimiter);
+
+            String daysSinceUpdated = discNoteBean.getDays()+"";
+            writer.append(daysSinceUpdated.equals("0") ? "" : daysSinceUpdated);
+            writer.append(delimiter);
+        } else {
+            writer.append(delimiter);
+            writer.append(delimiter);
+        }
+
+        if (discNoteBean.getDisType() != null)  {
+            writer.append(discNoteBean.getDisType().getName());
+            writer.append(delimiter);
+        }
+
+        writer.append(RESOLUTION_STATUS_MAP.get(discNoteBean.getResolutionStatusId())+"");
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getEventName());
+        writer.append(delimiter);
+
+        String eventOccurrence = null != discNoteBean.getStudyEventDefinitionBean()
+                && discNoteBean.getStudyEventDefinitionBean().isRepeating()
+                ? String.valueOf(discNoteBean.getEvent().getSampleOrdinal()) : "";
+        writer.append(eventOccurrence);
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getCrfName());
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getCrfStatus());
+        writer.append(delimiter);
+
+        String itemGroupName = discNoteBean.getItemGroupName() == null
+                ? "" : String.valueOf(discNoteBean.getItemGroupName());
+        writer.append(itemGroupName);
+        writer.append(delimiter);
+
+        String itemDataOccurence = discNoteBean.getItemDataOrdinal() == null
+                ? "" : String.valueOf(discNoteBean.getItemDataOrdinal());
+        writer.append(itemDataOccurence);
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getEntityName());
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getEntityValue());
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getDescription()+"");
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getDetailedNotes()+"");
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getAssignedUser().getName());
+        writer.append(delimiter);
+
+        writer.append(discNoteBean.getStudyId()+"");
+
+
+        writer.append("\n");
+        return writer.toString();
+
+
+    }
+
 
     public String serializeToString(EntityBean bean, boolean includeHeaderRow,
```

Comment:
add the delimiter as argument here instead of the method above

Reply:
done -- please see changes in new PR:
https://github.com/OpenClinica/OpenClinica/pull/3463